specification/columns/effectivecost.md

```diff
@@ -13,6 +13,8 @@ In cases where the [ChargeCategory](#chargecategory) is not "Usage" or "Purchase
 
 * The EffectiveCost MUST be calculated based on the EffectiveCost of the related charges if the charge is calculated based on other charges (e.g. [ChargeCategory](#chargecategory) is "Tax").
 * The EffectiveCost MUST match the [BilledCost](#billedcost) if the charge is unrelated to other charges (e.g. [ChargeCategory](#chargecategory) is "Credit").
+* The EffectiveCost MUST be the total cost unamortized by the corresponding *commitment discount* for the given charge period when [CommitmentDiscountStatus](#commitment-discount-status) is "Unused".
+* The EffectiveCost MUST be the total cost unamortized by a *negotiated discount* at the end of its [*term*](#glossary:term) and allocated to the last eligible *billing period*.
```

Comment:
I'm not sure what these are saying 😵‍💫

Reply:
"unamortized" --> "unallocated to a metered resource"?

Reply:
I suggest we omit these two requirements.

Reasoning:

- The Discount Handling attribute already describes commitment discounts in detail, so I’m not sure if it's necessary to highlight the unused-related requirement here. If we decide to include it, should we also add it to the other cost columns?
- Regarding negotiated discounts, we should specify that this only applies to those with burn-down commitments and not build-up commitments (e.g., Azure MACC, AWS EDP). When and if we decide to describe various types of negotiated discounts (looking forward to this! :)), I would prefer to do that within the Discount Handling attribute (as it applies to all columns) rather than in one of the cost columns.

Reply:
Line 16: Altered with Michael's suggestion.
Line 17: Removed